script.js

```diff
@@ -1 +1,152 @@
+//yehey 하이
+const todoForm = document.querySelector(".submitContainer"),
+    todoInput = todoForm.querySelector("input"),
+    todoList = document.querySelector(".todoListContainer"),
+    completeList = document.querySelector(".completeContainer");
 
+const TODOS_LS = "todos";
+let todos =[];    //여러개의 할 일들을 담을 배열
+const COM_LS = 'done';
+let done=[];
+
+function loadToDos(){
+    const localTodo = localStorage.getItem(TODOS_LS);
+    const localDone = localStorage.getItem(COM_LS);
+    if(localTodo!==null){
+        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
+        const local = JSON.parse(localTodo);
+        local.forEach(function(toDo){ addToDo(toDo.text);})
+    }
+    if(localDone!==null){
+        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
+        const local = JSON.parse(localDone);
+        local.forEach(function(toDo){ addComplete(toDo.text);})
+    }
```

Comment:
값이 있다면 todolist에 추가해라!라는 함수를 실행하는 것이니 이렇게 표현해도 좋을 것 같아요~
그치만 맨 위에 적었던 것처럼 하나의 객체로 묶으신다면 if문 하나로 해결 될 것 같습니다. 나중에 시간 되시면 바꿔보세요~~
```suggestion
    if(localTodo){
        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
        const local = JSON.parse(localTodo);
        local.forEach(function(toDo){ addToDo(toDo.text);})
    }
    if(localDone){
        //로컬스토리지에서 데이터 가져오기 String -> Obj 변환 필요
        const local = JSON.parse(localDone);
        local.forEach(function(toDo){ addComplete(toDo.text);})
    }
```